### ISSUE TITLE
Subnet definition does not match project

### ISSUE THREAD

**Issue by fmms** [Azure/data-management-zone#261]
Hi,

in the bastion template we have:
https://github.com/Azure/data-management-zone/blob/73e450a66062bafb6dc4bf3a1fef24cc4803c319/docs/reference/bastionhost/main.bicep#L31

However, in the configuration at https://github.com/Azure/data-management-zone/blob/73e450a66062bafb6dc4bf3a1fef24cc4803c319/infra/params.dev.json#L21 
an address range is referenced that does not fit to this default IP address.

As the bastion host is stored in this repository to be instantiated as part of the mangement zone i suggest to get that aligned.

regards

**Comment by fmms** [comment]
Unfortunately linked to https://github.com/Azure/azure-quickstart-templates/issues/2786 as well, if the solution should be redeployable.

**Comment by marvinbuss** [comment]
Thanks @fmms for submitting this. 

We are calling out [here](https://github.com/Azure/data-management-zone/blob/main/docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md#deployment) that this can be deployed to a Landing Zone or Management Zone. The default subnet address prefixes are actually within the address space of the default Data Landing Zone values. Hence, this will most likely stay as it is. Users can update the address space accordingly to land this into their management zone.

https://github.com/Azure/azure-quickstart-templates/issues/2786 is a well known issue that is not in control of our team. Hence, we also have to live with that network RP design. If you want to make it idempotent, you will have to add the Bastion Host template to the landing zone or management zone setup. Currently, we are evaluating whether we should do this. Please add your comment here, if you want us to work on this: https://github.com/Azure/data-landing-zone/issues/202
Reason why we have not done this so far is that most customers do not require a Bastion Host, because they already have a VPN or ExpressRoute connectivity that allows them to connect to services privately. Adding a Bastion host setup in those scenarios would just consume additional address space and is therefore not desired.

**Comment by fmms** [comment]
@marvinbuss yes, i did read that it can be instantiated in both, just had the feeling as this is stored in the data-management-zone repo, it should be by default aligned to that and not the data-landing-zone. Moreover, to me it seemed like logically this should be part of the management zone as this is an overarching service while testing this and not something to be used in production scenarios where you will have VNet peering.

**Comment by marvinbuss** [comment]
Due to the IAM requirements described [here](https://docs.microsoft.com/en-us/azure/bastion/vnet-peering#permissions) Azure Bastion is not necessarily an overarching service that is shared across all spokes. Due to these restrictions, users often deploy this into the respective spoke rather than use it as a shared resource. As a result, we are not dictating the use and deployment only in the Management Zone or Landing Zone. It can be landed into each one of them depending on the user group.

**Comment by marvinbuss** [comment]
I will convert this into a discussion for now. If changes are required based on the discussion, we may open a new issue.
